### ISSUE TITLE
[Ban Appeal] Ban Appeal (Again)

### ISSUE THREAD

**Issue by Austinator** [ObliqueNET/Server#1079]
Hello person reading this,

I was banned approximately 400 days ago, on I believe March 14th, 2020. I had been continuously disrespecting both the staff team and the general player base for quite some time prior to that. Whether or not I had been ill-mannered in a joking matter, I should have eased up a lot. I apologize to the people who I have disrespected, wittingly or not. I know most of the staff was pretty firm on deciding to not unban me, but I believe I deserve another chance over a year later.

My Discord is Bob Bobington#4577 if you feel the need to contact me there, and my MC Username was changed to BobingtonVEVO if you so generously decide to revoke my ban. I would prefer further discussion if you are to deny my appeal once again, if that's acceptable.

**Comment by Rasgnarok** [comment]
Honestly, I am just curious as to why you want to be unbanned. You previously said you wouldn't bother appealing because you're not interested in playing either way, and you know full well the amount of trolling we put up with, as well as the amount of times I forgave it.

**Comment by Austinator** [comment]
> Honestly, I am just curious as to why you want to be unbanned. You previously said you wouldn't bother appealing because you're not interested in playing either way, and you know full well the amount of trolling we put up with, as well as the amount of times I forgave it.

To be fully honest I didn't really want to come back because of a few staff members, who had some really vocal disliking for me at the time I had played before. A couple were pretty openly rude to me for reasons I can't really remember, gonna assume it was because of my rudeness, which didn't really justify it either way in my opinion. They kind of deterred me from wanting to play more since I figure they would likely treat me nearly the exact same as when I had been banned. Either way, they were the only thing wanting me to not appeal, I think I'm done being upset with them, why not appeal again? I'm also kinda curious if they can hold grudges for this long.

**Comment by Rasgnarok** [comment]
I am not sure I understand - you think people were rude to you because you were rude, and that's wrong because they're staff? There were only so many second chances we could give you, and staff clearly ran out of patience after I protected you countlessly. Sure, they have more patience as staff, but they are still human. You taunted my team, which was not the best use of the opportunities for redemption I gave you. I am not prepared to give you yet another one. Your ban should remain, since you don't seem to understand you picked the fights that led you to be banned.

**Comment by Austinator** [comment]
No no, I think I may have typed that a bit weird, sorry. I assume people were rude to me because I was rude, and that's wrong in my opinion because retaliation isn't really the best course of action. It does make sense that the more constant tormenting would have whittled their patience down to the point where they actually did retaliate, I think I acknowledged that. I 100% do realize that I had started everything that led to me being banned, I think either you misinterpreted what I'm trying to say or I'm having trouble getting my words out in a way that makes sense to you. I completely understand and accept that I was the overall cause of this, I was just pointing out that although I can change my behavior over the time of my absence, I don't know if the staff I had previously taunted will believe that. I don't know if I'm saying this right; I know that I was uncool to people and in return they were somewhat rude to me, I'm prepared to behave more appropriately but I don't know if people will forgive my past actions or if they'll immediately hate me if I had been unbanned.
